### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by morenol**
Fixes #

### Checklist
- [ ] My branch is up-to-date with development branch.
- [ ] Everything works and tested on latest stable Rust.
- [ ] Coverage and Linting have been applied

### CODE REVIEW

File: src/dataset/mod.rs [hunk 1]
```diff
@@ -8,9 +8,8 @@ pub mod digits;
 pub mod generator;
 pub mod iris;
 
-use crate::numbers::basenum::Number;
 #[cfg(not(target_arch = "wasm32"))]
-use crate::numbers::realnum::RealNumber;
+use crate::numbers::{basenum::Number, realnum::RealNumber};
```
Comment (morenol) [inline src/dataset/mod.rs:12]: This is just a change to remove compilation warning about unused code in wasm32

File: src/naive_bayes/multinomial.rs [hunk 2]
```diff
@@ -389,7 +389,6 @@ mod tests {
         assert!(iter.next().is_none());
     }
 
-    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
```
Comment (morenol) [inline src/naive_bayes/multinomial.rs:392]: my bad, I have to add this again

File: src/linear/logistic_regression.rs [hunk 3]
```diff
@@ -715,7 +715,7 @@ mod tests {
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
     #[test]
     fn lr_fit_predict() {
-        let x = DenseMatrix::from_2d_array(&[
+        let x: DenseMatrix<f64> = DenseMatrix::from_2d_array(&[
```
Comment (morenol) [inline src/linear/logistic_regression.rs:718]: using f64 instead of f32 for better precision in wasm32 tests

Comment (Mec-iS) [reply src/linear/logistic_regression.rs:718]: probably f64 is more stable but possibly less supported

**Pull request merged**